### ISSUE TITLE
Events › Coimbra › Convento de São Francisco

### ISSUE THREAD

**Issue by knoan** [ec2u/data#22]

**Comment by knoan** [comment]
* Events are already reported in detail by [Coimbra City Council / CoimbrAgenda](https://github.com/ec2u/data/blob/main/src/main/java/eu/ec2u/data/tasks/events/EventsCoimbraCity.md)
* Direct JSON-LD integration blocked by syntax errors (see [EventsCoimbraSaoFrancisco.md](https://github.com/ec2u/data/blob/main/src/main/java/eu/ec2u/data/tasks/events/EventsCoimbraSaoFrancisco.md))
* Direct RSS integration not worth, as it provides less details than _Coimbra City Council / CoimbrAgenda_

@ec2u/mmt

**Comment by knoan** [comment]
Superseded by focusing on university sources.
